sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.1.0-preview.2 (Unreleased)
+## 0.1.0-preview.2
```

Comment:
I think you're supposed to put the release date in the parens like `## 0.1.0-preview.2 (2020-07-02)`?